axelor-account/src/main/resources/views/Invoice.xml

```diff
@@ -1099,6 +1099,18 @@ InvoiceSet.length > 0))"
 		<call class="com.axelor.apps.account.web.InvoiceController" method="checkPartnerBankDetailsList"/>
 	</action-method>
 
+	<action-group name="action-invoice-group-invoice-ventilate-onclick">
+		<action name="save"/>
+		<action name="action-invoice-validate-analytical-distribution-template-prompt" 
+			if="invoiceLineList.find{it->!it.account.analyticDistributionAuthorized &amp;&amp; it.analyticDistributionTemplate != null} != null"/>
```

Comment:
Don't create action-validate but throw AxelorException on 'ventilate' method of InvoiceServiceImpl